templates/zerver/api/changelog.md

```diff
@@ -68,6 +68,15 @@ format used by the Zulip server that they are interacting with.
 Feature levels 123-124 are reserved for future use in 5.x maintenance
 releases.
 
+**Feature level 125**
+
+* [`GET /events`](/api/get-events): Added `user_topic` event type
+  which is sent when a topic is muted or unmuted. When a user either
+  requests for `user_topic` event only or requests for both `muted_topics`
+  and `user_topic` event, only `user_topic` events are sent.
```

Comment:
I feel like the information about `muted_topics` could be a separate bullet point, for clarity and since it impacts both the `GET /events` and `POST /register` endpoints return values? Something like ...

> - `GET /events`, `POST /register`: `muted_topics` array is no longer sent when `user_topic` is specified as an event type.

And this information should be added to the `muted_topics` array descriptions for both endpoints (with a feature level note).

Reply:
I'd add this bullet point third.